docs/appendices/release-notes/5.5.0.rst

```diff
@@ -54,6 +54,9 @@ Breaking Changes
 - `UPDATE`` statements will continue on row failures instead of always showing 
   an error. The affected rows are displayed by the resulting row count.
 
+- Restricted :ref:`Logical Replication <administration-logical-replication>` of
+  tables created on a cluster with higher version to a cluster with lower version.
+
```

Comment:
There are some actually compatible versions where we didn't have indexing/metadata changes. If users used to do such replication, it's a breaking change for them.